src/components/Footer.tsx

```diff
@@ -0,0 +1,142 @@
+import EmailIcon from '@mui/icons-material/Email';
+import GitHubIcon from '@mui/icons-material/GitHub';
+import InstagramIcon from '@mui/icons-material/Instagram';
+import LinkedInIcon from '@mui/icons-material/LinkedIn';
+import TwitterIcon from '@mui/icons-material/Twitter';
+import { Box, Typography, Link, Icon, useMediaQuery } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+
+import { GradientBorderDivider } from '@components';
+import { FooterResponse, Network } from '@utils/types';
+import theme from 'theme';
+
+import SlackIcon from '../../public/icons/slack-icon.svg';
+import { addLineBreakAfterPeriod } from '@utils/helpers';
+
+const iconNetworkMapper = {
```

Comment:
would be better to move this to another place, because I think this mapper will be reused in many places.